step-plans/step-plans-core/src/main/java/step/core/plans/PlanAccessorImpl.java

```diff
@@ -29,6 +29,11 @@ public class PlanAccessorImpl extends AbstractAccessor<Plan> implements PlanAcce
 
 	public PlanAccessorImpl(Collection<Plan> collectionDriver) {
 		super(collectionDriver);
+		createIndexIfNeeded();
+	}
+
+	private void createIndexIfNeeded() {
+		createOrUpdateIndex("visible");

```

Comment:
At some point we should somehow use constants or add a comment to the corresponding field as it can easily happen that we rename the field and the index wouldn't work anymore